next-app/app/page.js

```diff
@@ -37,24 +134,25 @@ export default function Home() {
 }
 
 // Displays the weather data in a more readable state
-function WeatherData({ apiData }) {
-  // Seperate the API data from today and the rest of the week
-  const todayData = apiData[0];
-  const future = apiData.slice(1, 7);
-  const [marked, setMarked] = useState(false);
-  const markDay = marked ? 'marked' : '';
-
-  let weekday = {
-    0: 'Monday',
-    1: 'Tuesday',
-    2: 'Wednesday',
-    3: 'Thursday',
-    4: 'Friday',
-    5: 'Saturday',
-    6: 'Sunday',
+function WeatherData({
+  weatherData,
+  addDate,
+  showToast,
+  setShowToast,
+  dateRequested,
+  dupe,
+  undoDate,
+}) {
+  const weekday = {
```

Comment:
all this static data can probably be taken out of components, bc it doesn't depend on any state